src/EPPlus/Drawing/ExcelDrawings.cs

```diff
@@ -1553,51 +1552,51 @@ public ExcelControlScrollBar AddScrollBarControl(string Name)
+        public ExcelOleObject AddOleObject(string name, string olePath)
         {
-            return AddOleObject(Name, OlePath, false, false, null);
+            return AddOleObject(name, olePath, false, false, null);
         }
 
         /// <summary>
         /// Embed or link an OLE object using a string file path.
         /// </summary>
-        /// <param name="Name">Name of the drawing.</param>
-        /// <param name="OlePath">The path to the file.</param>
-        /// <param name="LinkToFile">Optional - True: File will be linked. False: File will be embedded.</param>
-        /// <param name="DisplayAsIcon">Optional - Set to display the object as in icon.</param>
-        /// <param name="IconFilePath">Optional: Path for the icon.</param>
+        /// <param name="name">Name of the drawing.</param>
+        /// <param name="olePath">The path to the file.</param>
+        /// <param name="linkToFile">Optional - True: File will be linked. False: File will be embedded.</param>
+        /// <param name="displayAsIcon">Optional - Set to display the object as in icon.</param>
+        /// <param name="iconFilePath">Optional: Path for the icon.</param>
         /// <returns>A new drawing of type ExcelOleObject.</returns>
-        public ExcelOleObject AddOleObject(string Name, string OlePath, bool LinkToFile = false, bool DisplayAsIcon = false, string IconFilePath = null)
+        public ExcelOleObject AddOleObject(string name, string olePath, bool linkToFile = false, bool displayAsIcon = false, string iconFilePath = null)
         {
             ExcelOleObjectParameters parameters = new ExcelOleObjectParameters()
             {
-                LinkToFile = LinkToFile,
-                DisplayAsIcon = DisplayAsIcon,
-                OlePath = OlePath,
+                LinkToFile = linkToFile,
+                DisplayAsIcon = displayAsIcon,
+                OlePath = olePath,
             };
-            return AddOleObject(Name, OlePath, parameters, IconFilePath);
+            return AddOleObject(name, olePath, parameters, iconFilePath);
         }
 
         /// <summary>
         /// Embed or link an OLE object using a string file path.
         /// </summary>
-        /// <param name="Name">Name of the drawing.</param>
-        /// <param name="OlePath">The path to the file.</param>
-        /// <param name="Parameters">Object containing additional parameters.</param>
-        /// <param name="IconFilePath">Optional: Path for the icon.</param>
+        /// <param name="name">Name of the drawing.</param>
+        /// <param name="olePath">The path to the file.</param>
+        /// <param name="parameters">Object containing additional parameters.</param>
+        /// <param name="iconFilePath">Optional: Path for the icon.</param>
         /// <returns>A new drawing of type ExcelOleObject.</returns>
         /// <exception cref="Exception">Can Throw exception if Name exsists.</exception>
```

Comment:
Eftersom `ExcelOleObjectParameters` verkar innehålla alla parametrar utom `Name` blir det redundant att ha den som ett av argumenten tillsammans med de övriga.  Det möjliggör t ex att man gör så här:

```csharp
var parameters = new ExcelOleObjectParameters
{
    OlePath = "c:\file1.pdf",
    DisplayAsIcon = false
}

sheet.AddOleObject("name", "c:\file2.pdf", parameters, true);
```
...d v s funktionen möjliggör att man kan skicka in konflikterande parametrar.

Mitt förslag är att man tar ner denna version av funktionen till följande:

```csharp
public ExcelOleObject(string name, Action<ExcelOleObjectParameters> configHandler)
{
    var params = new ExcelOleObjectParameters();
    configHandler.Invoke(params);
    // fortsätt funktionen...
}
```